cfgov/v1/wagtail_hooks.py

```diff
@@ -49,6 +49,17 @@ def share(page, is_sharing, is_live):
     page.save()
 
 
+@hooks.register('insert_editor_js')
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/2344/commits/362e1236b35477c0095fd8ff5955e2e504c4c342.